packages/material-ui-lab/src/SliderStyled/SliderStyled.d.ts

```diff
@@ -48,4 +48,46 @@ export const SliderValueLabel: React.FC<SliderValueLabel>;
  */
 declare const Slider: ExtendSliderUnstyled<SliderStyledTypeMap>;
 
+export type SliderStyledClassKey =
+  /** Class name applied to the root element. */
+  | 'root'
+  /** Class name applied to the root element if `color="primary"`. */
+  | 'colorPrimary'
+  /** Class name applied to the root element if `color="secondary"`. */
+  | 'colorSecondary'
+  /** Class name applied to the root element if `marks` is provided with at least one label. */
+  | 'marked'
+  /** Pseudo-class applied to the root element if `orientation="vertical"`. */
```

Comment:
Considering the CSS is of specificity 1 for this, I think that we consider it a class name, also because the generated class name doesn't follow the convention for pseudo-classes.

```suggestion
  /** Class name applied to the root element if `orientation="vertical"`. */
```